packages/app/src/cli/models/extensions/function-specifications/delivery_customization.ts

```diff
@@ -0,0 +1,11 @@
+import {createFunctionSpec} from '../functions.js'
+
+const spec = createFunctionSpec({
+  identifier: 'delivery_customization',
+  externalType: 'delivery_customization',
+  externalName: 'Delivery customization',
+  public: false,
+  templatePath: (lang) => `checkout/${lang}/delivery-customization/defaultt`,
```

Comment:
This typo is causing an empty folder to be generated for Delivery Customization function extensions

Reply:
Thanks! It will be fixed in https://github.com/Shopify/cli/pull/922